### PR TITLE
Table consistency tickets 

### PR DESCRIPTION
**Pull request by tiffwong**
## Description

Make sure all tables are referencing the right `TableRow` component and define the height for table rows

## How to test

Check the following tables to make sure the color specs and row height (headers are 46px, rows are 40px) are consistent in dark mode 
- Linode Details Activity Feed table
- Invoice table

Check the header hover color spec for the Linodes and StackScripts Landing table in dark mode

### CODE REVIEW

File: packages/manager/src/components/EntityTable/GroupedEntitiesByTag.tsx [hunk 1]
```diff
@@ -11,16 +10,13 @@ import PaginationFooter, {
...
 import Table from 'src/components/Table';
 import TableCell from 'src/components/TableCell';
+import TableRow from 'src/components/TableRow';
 import { useInfinitePageSize } from 'src/hooks/useInfinitePageSize';
 import { groupByTags, sortGroups } from 'src/utilities/groupByTags';
 import EntityTableHeader from './EntityTableHeader';
 import { ListProps } from './types';
 
 const useStyles = makeStyles((theme: Theme) => ({
-  root: {},
```
Comment (tiffwong) [inline packages/manager/src/components/EntityTable/GroupedEntitiesByTag.tsx:20]: These classes aren't being used

File: packages/manager/src/features/Volumes/ListGroupedVolumes.tsx [hunk 2]
```diff
@@ -113,43 +113,41 @@ const ListGroupedVolumes: React.FC<CombinedProps> = (props) => {
                 count,
               }) => {
                 return (
-                  <React.Fragment>
```
Comment (tiffwong) [inline packages/manager/src/features/Volumes/ListGroupedVolumes.tsx:116]: Only change here was getting rid of the `<React.Fragment>`

File: packages/manager/src/features/linodes/LinodesDetail/LinodeAdvanced/LinodeConfigs.tsx [hunk 3]
```diff
@@ -56,9 +56,6 @@ type ClassNames =
   | 'addNewWrapper'
   | 'tableCell'
   | 'labelColumn'
-  | 'vmColumn'
```
Comment (tiffwong) [inline packages/manager/src/features/linodes/LinodesDetail/LinodeAdvanced/LinodeConfigs.tsx:59]: These weren't being used anywhere

File: packages/manager/src/themes.ts [hunk 4]
```diff
@@ -117,6 +117,15 @@ const genericStatusPillStyle = {
   },
 };
 
+const genericTableHeaderStyle = {
```
Comment (tiffwong) [inline packages/manager/src/themes.ts:120]: Added this to dark mode which solves the inconsistent hover state for dark mode

**Review by codyfinn — APPROVED** [review]
Looks good to me!

**Review by bnussman — APPROVED** [review]

**Review by HanaXu — COMMENTED** [review]
Not sure why the padding right is currently removed, but it's causing the last cell to look a little tight in the invoice table.
 
Screen Shot 2021-07-07 at 3 23 35 PM

The StackScripts table border top & bottom is 1 px here but 2 px everywhere else

Screen Shot 2021-07-07 at 3 31 49 PM
Screen Shot 2021-07-07 at 3 32 24 PM

This was probably my bad, but can we also fix the dark mode `upgrade` chip hover in Kubernetes? 😅 

Screen Shot 2021-07-07 at 3 27 24 PM

File: packages/manager/src/features/Billing/InvoiceDetail/InvoiceTable.tsx [hunk 5]
```diff
@@ -21,6 +21,9 @@ const useStyles = makeStyles((theme: Theme) => ({
     border: `1px solid ${theme.cmrBorderColors.borderTable}`,
     '& thead th': {
       borderBottom: `1px solid ${theme.cmrBorderColors.borderTable}`,
+      '&:last-of-type': {
+        paddingRight: 10,
```
Comment (HanaXu) [inline packages/manager/src/features/Billing/InvoiceDetail/InvoiceTable.tsx:25]: This should be 15, but LGTM otherwise!

**Review by HanaXu — APPROVED** [review]

**Pull request merged**